model/prelude_mem_addrtype.sail

```diff
@@ -0,0 +1,18 @@
+/*=======================================================================================*/
+/*  This Sail RISC-V architecture model, comprising all files and                        */
+/*  directories except where otherwise noted is subject the BSD                          */
+/*  two-clause license in the LICENSE file.                                              */
+/*                                                                                       */
+/*  SPDX-License-Identifier: BSD-2-Clause                                                */
+/*=======================================================================================*/
+
+/* Note: virtaddr includes physical addresses when address translation is disabled.
```

Comment:
As noted below, I don't see how this can work once physaddr is actually `bits(34)` on RV32. But I guess that can be fixed once this is actually changed.

Reply:
Well with satp.MODE = Bare you can't address more than XLEN bits of PA space, you have to have translation for that